.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Do we still need that file with prettier?

Reply:
@popojargo the point of .editorconfig is to specify a common editor configuration for all developers working on the same project. EditorConfig is supported by many editors and is thus widely used. You don't have to duplicate overlapping settings between the two since editorconfig will work jointly with prettier.

The editor can use prettier to reformat a file at the point configured (aka. for this project on staged), but I don't think that prettier can be used by the editor each time it is opening a new line, so the editor cannot rely entirely on prettier for formatting thus the use of .editorconfig. Prettier is a plugin while the config is more "native".

The .editorconfig is also required for GitHub code views to render your tabs (not spaces) with any desired unit of spaces instead of the 8 that is default, so even a code base that is not shared between multiple developers may still need to define this file to look "prettier" on GitHub.